htdocs/api/class/api_setup.class.php

```diff
@@ -911,7 +911,7 @@ public function getListOfTowns($sortfield = "zip,town", $sortorder = 'ASC', $lim
      * @throws 400 RestException
      * @throws 200 OK
      */
-    public function getPaymentTerms($sortfield = "sortorder", $sortorder = 'ASC', $limit = 100, $page = 0, $active = 1, $sqlfilters = '')
+    public function getPaymentTerms($sortfield = "code", $sortorder = 'ASC', $limit = 100, $page = 0, $active = 1, $sqlfilters = '')
```

Comment:
Why no more sorting on the default sorting field that is dedicated to define the sort order of element in dictionary ?

Reply:
@eldy oups i gorget this field. I fixed it